packages/protocol-kit/src/adapters/web3/contracts/GnosisSafe/GnosisSafeContractWeb3.ts

```diff
@@ -6,10 +6,10 @@ import {
   SafeVersion,
   SafeSetupConfig
 } from '@safe-global/safe-core-sdk-types'
-import { Gnosis_safe as GnosisSafe_V1_0_0 } from '../../../typechain/src/web3-v1/v1.0.0/Gnosis_safe'
-import { Gnosis_safe as GnosisSafe_V1_1_1 } from '../../../typechain/src/web3-v1/v1.1.1/Gnosis_safe'
-import { Gnosis_safe as GnosisSafe_V1_2_0 } from '../../../typechain/src/web3-v1/v1.2.0/Gnosis_safe'
-import { Gnosis_safe as GnosisSafe_V1_3_0 } from '../../../typechain/src/web3-v1/v1.3.0/Gnosis_safe'
+import { Gnosis_safe as GnosisSafe_V1_0_0 } from '../../../../../typechain/src/web3-v1/v1.0.0/Gnosis_safe'
+import { Gnosis_safe as GnosisSafe_V1_1_1 } from '../../../../../typechain/src/web3-v1/v1.1.1/Gnosis_safe'
+import { Gnosis_safe as GnosisSafe_V1_2_0 } from '../../../../../typechain/src/web3-v1/v1.2.0/Gnosis_safe'
+import { Gnosis_safe as GnosisSafe_V1_3_0 } from '../../../../../typechain/src/web3-v1/v1.3.0/Gnosis_safe'
```

Comment:
We need to setup root level routes for this packages. Too much path navigations in these files.

I would like it as a separate PR

Reply:
yes, all the repo should have root level routing